roles/auth_source_ldaps/README.md

```diff
@@ -0,0 +1,77 @@
+theforeman.foreman.auth_source_ldaps
```

Comment:
I think the only thing in this PR that prevents me from just merging it is the name of the role…

We have been using the plural for of the module name for most roles, to denote that the role is capable of managing multiple instances of $thing, but `auth_source_ldaps` totally weirds me out.

You can sell me `auth_sources`, or `auth_sources_ldap`, but not `auth_source_ldaps` ;)

Reply:
Haha, yeah I was actually going with `auth_sources_ldap` first at the customer but when I made this PR I noticed that the API endpoint in Foreman is actually called `auth_source_ldaps`.

But I will happily change the name, cause it kinda bugs me too. Just note that it will make the role differ from the API endpoint.

Reply:
To hell with the API endpoint names :)

(you can quote me on that in any Foreman core meeting!)